kedro/io/core.py

```diff
@@ -490,10 +503,19 @@ class AbstractVersionedDataSet(AbstractDataSet, abc.ABC):
         >>>
         >>>     def _exists(self) -> bool:
         >>>         path = self._get_load_path()
-        >>>         return path.is_file()
+        >>>         return Path(path).exists()
         >>>
         >>>     def _describe(self):
         >>>         return dict(version=self._version, param1=self._param1, param2=self._param2)
+
+    Example catalog.yml specification
+    ::
+        my_dataset:
```

Comment:
similar re: rendering in docs and blank line between `::` and `my_dataset`.

Reply:
I think I got it fixed.  popped it into an online renderer and it seemed to like the latest change.